<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QMainWindow" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>1262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menuWindow {
background:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QLineEdit {
padding: 2px;
border-style: solid;
border:2px solid pink ;

border-radius: 8px;


}

QDateEdit 
{
padding: 1px;
border-style: solid;
border: 2px solid pink;
border-radius: 8px;
}


QPushButton {
color: white;

background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 13px;
padding-left: 5px;
padding-right: 5px;

}


QLabel {
color :  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
font-weight: bold;
font-size: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
background-color: transparent;
border-image: url(:/img/img/retour.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_11 {
background-color: transparent;
border-image: url(:/img/img/espace admin.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
background-color: transparent;
border-image: url(:/img/img/ajouter.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>352</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3{
background-color: transparent;
border-image: url(:/img/img/chercher.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5{
background-color: transparent;
border-image: url(:/img/img/chercher.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 18px;
}</string>
    </property>
    <property name="text">
     <string>   Note Menu</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>831</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QLineEdit" name="lineEdit_repas">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>157</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dessert">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>197</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>157</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=white&gt; Repas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>197</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=white&gt; Dessert
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>117</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=white&gt; Jour</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_date2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>237</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>77</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;font color=white&gt; Identifiant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer_menu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>317</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>77</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>287</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_jours">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>117</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;font color=white&gt;Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_menu">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>441</width>
        <height>561</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_menu2_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_menu</zorder>
     <zorder>lineEdit_repas</zorder>
     <zorder>lineEdit_dessert</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>label_2</zorder>
     <zorder>dateEdit_date2</zorder>
     <zorder>label</zorder>
     <zorder>imprimer_menu</zorder>
     <zorder>lineEdit_id</zorder>
     <zorder>ajouter</zorder>
     <zorder>lineEdit_jours</zorder>
     <zorder>label_6</zorder>
     <zorder>label_menu2_2</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>360</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nv repas :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nvdessert">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>390</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>390</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nv dessert :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_nvdate">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>420</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chercher_plat">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>123</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher par repas</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabMenu">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>541</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_menu">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>460</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>40</y>
        <width>131</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>trier</string>
      </property>
      <widget class="QPushButton" name="pushButton_trierParId">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>21</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string> par id</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_trierParJours">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>50</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string> par jours</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>150</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>supprimer</string>
      </property>
      <widget class="QPushButton" name="pb_supprimer_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>67</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_afficher_par_typpe">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>280</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher par jours</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_chercherMenu">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Chercher par id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nv date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>330</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nv jours : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nvjours">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nvrepas">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>360</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercherMenu_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>126</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_chercherMenu">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>126</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_afficher_note">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>411</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_toff">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>771</width>
        <height>541</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableView" name="table_note">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>521</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher nbre de ligne + sommes des notes + moyenne des notes + min note+ max note</string>
      </property>
     </widget>
     <zorder>label_toff</zorder>
     <zorder>pushButton_afficher_note</zorder>
     <zorder>table_note</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
   </widget>
   <zorder>label_11</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1272</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
